docs/modules/integrate/pages/kafka-connector.adoc

```diff
@@ -103,6 +103,25 @@ properties.put("schema.registry.url", schemaRegistryUrl);
 Keep in mind that once the record deserialized, Jet still needs to know
 how to serialize/deserialize the record internally.
 
+== Embedded Avro Schema
+It is possible to employ an embedded Avro schema for GeneralRecord by providing the schema as a parameter for Kafka producer/consumer, along with a HazelcastAvro serialization and deserialization class.
```

Comment:
Do you mean `GenericRecord`? If not, I cannot find any information on `GeneralRecord` - not in Hazelcast nor in Avro docs/discussions - can you point me at anything?
Is HazelcastAvro intentionally a single word?

Leaving those questions aside, I'd recommend rewording to:

To use an embedded Avro schema for `GeneralRecord`, you must provide the following as parameters for the Kafka producer and/or consumer:

* The schema
* A HazelcastAvro serialization class
* A HazelcastAvro deserialization class

Reply:
fixed